### PR TITLE
Modified email to remove references to the feedback feature

### LINKED ISSUE #1279: Welcome email to rephrase
Welcome email references 'Send Feedback' interaction - a feature available on the staging sites but not in production. This message is still appearing in production's Welcome emails.
Recommendation: Remove this line from the welcome email, because the Send Feedback feature doesn't exist.
Device: All 
Where: Welcome email
image

### PR DESCRIPTION
**Pull request by hughbric**
#### What? Why?

Closes #1279

Removes references the the 'Send Feedback' feature that does not exist.

#### What should we test?
signup_confirmation.en-GB.text.erb

#### Release notes
Removes references the the 'Send Feedback' feature that does not exist.

Changelog Category: Removed

### CODE REVIEW

**Comment by hughbric** [comment]
What? Why?
Closes #3509 

Add a button to the email footer (if instagram link exists)

What should we test?
Emails and their footers

Release notes
We did not add the link to the OFN instagram (can be added later) after the discussion with Pau. 
We created a new CSS class for the color of the instagram button 

Changelog Category: Add

File: app/views/spree/user_mailer/signup_confirmation.en-GB.text.erb [hunk 1]
```diff
@@ -4,11 +4,8 @@ Welcome to Open Food Network UK! Your login email is <%= @user.email %>
 
 You can go online and start shopping through food hubs and local producers you like at http://openfoodnetwork.org.uk
 
-We welcome all your questions and feedback; you can use the Send Feedback button on the site or email us at hello@openfoodnetwork.org.
```
Comment (kristinalim) [inline app/views/spree/user_mailer/signup_confirmation.en-GB.text.erb]: @hughbric Sorry for the confusion! You did as described in the issue description, but a later [comment](https://github.com/openfoodfoundation/openfoodnetwork/issues/1279#issuecomment-429145869) says to change the text instead:

> Update text to read: "We welcome all your questions and feedback; you can email us at hello@openfoodnetwork.org.au" <--- note that this email will be different for each instance.

Could you update?

For the email address specific to the instance, you can use `ContentConfig.footer_email`, which is set in: **Admin > Configuration > Content > Footer and External Links > Email**.

**Comment by RachL** [comment]
Actually @kristinalim requested changes here. She meant to put the PR back in dev but apparently there was an update issue. So I'm moving it back to in dev :)

**Comment by luisramos0** [comment]
hello @hughbric do you think you will be able to pick this up again?
let us know. thanks!

File: app/views/spree/user_mailer/signup_confirmation.en-GB.text.erb (hunk 1 above)
Comment (luisramos0) [reply app/views/spree/user_mailer/signup_confirmation.en-GB.text.erb]: I wonder what this en-GB specific file is, isnt this supposed to be in en-GB in some way?

**Comment by RachL** [comment]
@hughbric Hi! The issue you were working on has been dealt with in another way, see https://github.com/openfoodfoundation/openfoodnetwork/issues/1279#issuecomment-497498794 I'm closing this pull request.

Don't hesitate if you have any question.

**Pull request closed without merging**